book/website/community-handbook/consistency/consistency-formatting.md

```diff
@@ -0,0 +1,194 @@
+(ch-consistency-formatting)=
+# Formatting
+
+Formatting refers to how _The Turing Way_ book is written and directly affects the book's appearance and presentation.
+
+_The Turing Way_ is hosted online as a Jupyter Book and uses the formatting recommendations as described in their [documentation](https://jupyterbook.org/intro.html).
+Proper formatting ensures that _The Turing Way_ is readable, accessible, and resembles a modular piece of work.
+
+
+(ch-consistency-formatting-hr)=
+## Hard Requirements
+
+The hard requirements in the consistency checklist ensure that _The Turing Way_ prioritises accessibility, collaboration, readability and ease of use. 
+The checks that deal with the _The Turing Way's_ formatting include:
+
+(ch-consistency-formatting-hr-markdown)=
+### Check 1:  Use Markdown for creating your content
+
+_The Turing Way_ should be written in [Markdown](https://en.wikipedia.org/wiki/Markdown) where possible, so that the Jupyter Book renders as intended. 
+
+Some earlier chapters in _The Turing Way_ were not entirely written in Markdown, thereby making some of its content hard to read. 
```

Comment:
This is a bit contradictory because the second 'note' is about checklists written in Markdown, but still difficult to read/rendered incorrectly. So it's not just the issue of chapters not being written entirely in Markdown...

Reply:
True. In line 19, I snuck in a 'where possible' to imply that not writing Markdown is okay sometimes.

My intention with line 21 was to preface an example of how not using Markdown can cause problems,

I'll reword the sentence to reflect this